operator/src/main/java/org/bf2/operator/controllers/ManagedKafkaController.java

```diff
@@ -41,17 +35,14 @@
 import javax.inject.Inject;
 
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
 import java.util.Optional;
 
 @ControllerConfiguration(
```

Comment:
Also add the `ControllerEventFilter.class` as the `onUpdateFilter` on this annotation.